### DISCUSSION TITLE
How to block YouTube's home page but not video search?

### DISCUSSION THREAD

**Discussion by rhysday** [qutebrowser/qutebrowser#6785] [General]
I've tried blocking www.youtube.com (home page) but I still want to be able to search for videos. For example, could I still access https://www.youtube.com/feed/library ? but have youtube.com blocked?

**Comment by The-Compiler** [comment]
If you have the `adblock` library installed, you should be able to write an [adblock rule](https://adblockplus.org/filter-cheatsheet) doing so (and e.g. point `content.blocking.adblock.lists` to a `file://` URL containing it).
